src/parse.ts

```diff
@@ -373,7 +441,7 @@ function parseBasicSchema(details: BasicDescribe, settings: Settings, rootSchema
   if (rootSchema) {
     return makeTypeContentRoot({
       joinOperation: 'union',
-      children: [makeTypeContentChild({ content, interfaceOrTypeName, jsDoc })],
```

Comment:
There were some repeated `jsDoc` entries that ended up belonging to children when they should have not, because they were generating duplicate descriptions.